react_app/src/components/ChallengeCard.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+
+import Typography from '@material-ui/core/Typography'
+import Card from '@material-ui/core/Card'
+import CardActions from '@material-ui/core/CardActions'
+import CardContent from '@material-ui/core/CardContent'
+import Divider from '@material-ui/core/Divider'
+
+const useStyles = makeStyles(theme => ({
+  root: {
```

Comment:
I hope you're enjoying being able to re-use the word "root" here to specify your intention *knowing* you're not going to fuck up the rest of the app :) welcome to the future!